frontend/src/lib/taxonomy.tsx

```diff
@@ -510,97 +510,102 @@ export const CORE_FILTER_DEFINITIONS_BY_GROUP = {
         },
         // exception tracking
         $exception_list: {
-            label: 'Exception list',
-            description: 'List of one or more associated exceptions',
+            label: 'Exception List',
```

Comment:
Not sure if it applies here but I thought we prefer the use of sentence case: https://posthog.com/handbook/content-and-docs/posthog-style-guide#use-sentence-case-for-titles

Reply:
We have both in the taxonomy and most of them are capitalized. Might be a good opportunity to change that though 🤔 
<img width="681" alt="Screenshot 2025-04-03 at 16 07 05" src="https://github.com/user-attachments/assets/79969d6a-2cc0-46d7-8427-f8522569a2c9" />